cmd/ocm-support/patch/organizations/cmd.go

```diff
@@ -0,0 +1,103 @@
+package organizations
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/organization"
+	"github.com/openshift-online/ocm-support-cli/pkg/request"
+)
+
+var args struct {
+	filter     string
+	dryRun     bool
+	maxRecords int
+}
+
+// CmdPatchOrganizations represents the organizations patch command
+var CmdPatchOrganizations = &cobra.Command{
+	Use:     "organizations [id]",
+	Aliases: utils.Aliases["organizations"],
+	Short:   "Patches an organization for the given ID or organizations matching the filter passed",
+	Long:    "Patches an organization for the given ID or organizations matching the filter passed",
```

Comment:
nitpick: I think we can drop the word "passed" here (and in the pathSubscription command)